go-controller/pkg/ovn/secondary_layer2_network_controller_test.go

```diff
@@ -405,15 +407,12 @@ var _ = Describe("OVN Multi-Homed pod operations for layer2 network", func() {
 					Expect(ok).To(BeFalse())
 				}
 
-				Expect(fakeOvn.controller.WatchNamespaces()).To(Succeed())
-				Expect(fakeOvn.controller.WatchPods()).To(Succeed())
+				Expect(fakeOvn.networkManager.Start()).To(Succeed())
```

Comment:
defer a stop of the networkManager to prevent data races?

Reply:
good catch :)